docs/dev/new.rst

```diff
@@ -52,18 +52,18 @@ instances to be considered subclasses and instances of the protocol!
 As an example, let's look at the base :class:`~cosmology.api.Cosmology` protocol.
 
     >>> from dataclasses import dataclass
```

Comment:
Not sure about making this REPL-style (`>>>`), who would ever write all this interactively?

Reply:
With https://sybil.readthedocs.io/en/latest/rest.html#restructured-text-parsers these are now `code-block`, not REPL.